### ISSUE TITLE
Add colours for new games

### ISSUE THREAD

**Issue by YaManicKill** [porybox/porybox#665]
Sun, Moon, R, G, B, and Y all need colours.

**Comment by YaManicKill** [comment]
Added this to 1.4. Not because it shouldn't be in 1.3, but because it shouldn't be a focus. If someone gets it done after the other things are done, and before we release, that's fine. But we should focus on the other things, and if there are 0 bugs/regressions, and we've implemented everything in 1.3, we are releasing. It's not going to wait on this.

Any objections?

**Comment by YaManicKill** [comment]
I've moved this back to 1.3 because it would be nice to have it completed, but it's not going to be a blocker. If everything else is finished, and we have tested it properly, and not found any bugs (or all bugs have been fixed) then 1.3 still gets released.
